raster/r.proj/testsuite/test_rproj.py

```diff
@@ -0,0 +1,183 @@
+"""
+Name:       r.texture test
+Purpose:    Tests r.texture and its flags/options.
+
+Author:     Sunveer Singh, Google Code-in 2017
+            Chung-Yuan Liang, modified in 2024
+Copyright:  (C) 2017 by Sunveer Singh and the GRASS Development Team
+Licence:    This program is free software under the GNU General Public
+            License (>=v2). Read the file COPYING that comes with GRASS
+            for details.
+"""
+
+from grass.gunittest.case import TestCase
+from grass.gunittest.checkers import text_to_keyvalue
+from grass.gunittest.gmodules import call_module
+import os
+
+raster_info = """north=35.8096296297222
+south=35.6874074075
+east=-78.608
+west=-78.7746666666667
+nsres=0.000740740740740727
+ewres=0.000666666666666686
+rows=165
+cols=250
+cells=41250"""
+
+src_project = "nc_spm_full_v2alpha2"
+dst_project = "nc_latlong"
+
+
+class TestRasterreport(TestCase):
+    input = "elevation"
+
+    @classmethod
+    def setUpClass(cls):
+        cls.runModule("g.proj", project=dst_project, epsg="4326", flags="c")
+        cls.runModule("g.mapset", mapset="PERMANENT", project=dst_project)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.runModule("g.mapset", mapset="PERMANENT", project=src_project)
+        dbase = call_module("g.gisenv", get="GISDBASE")
+        os.system(f"rm -rf {dbase}/{dst_project}")
```

Comment:
Likely [shutils.rmtree](https://docs.python.org/3/library/shutil.html#shutil.rmtree) is what you need.

Off topic for this PR, but there is _create_project_ and _g.proj_, maybe there can be _delete_project_ too. Checking for locks would make sense.

Reply:
That's what I need. Thanks!